app/services/GovernmentSpendService.scala

```diff
@@ -42,7 +42,10 @@ class GovernmentSpendService @Inject()(
     taxIdentifier match {
       case Some(value) =>
         middleConnector.connectToGovernmentSpend(taxYear, value).map { response =>
-          response.json.as[Map[String, Double]].toSeq.sortBy(_._2).reverse
+          val sortedSpendData = response.json.as[Map[String, Double]].toList.sortBy(_._2).reverse
+          if (taxYear == 2018) {
+            GovernmentSpendUtils.reorderDataBasedOnCategories(sortedSpendData, "Culture", "Environment")
+          } else sortedSpendData
```

Comment:
Can you push the year into `reorderDataBasedOnCategories` and just return the real sortedSpendData? I think it would read better.